src/test/java/org/springframework/data/couchbase/domain/AirportRepository.java

```diff
@@ -37,4 +43,9 @@
 	@Query("#{#n1ql.selectEntity} where iata = $1")
 	List<Airport> getAllByIata(String iata);
 
+	long countByIataIn( String... iata );
```

Comment:
formatting: whitespace looks off inside the ()

Reply:
ok.